From: Sergey Senozhatsky <senozhatsky@chromium.org>
Subject: zram: fixup mark_slot_accessed()
Date: Thu, 15 Jan 2026 12:19:00 +0900

mark_slot_accessed() should continue using ktime_get_boottime_seconds()

Link: https://lkml.kernel.org/r/20260115031922.3813659-1-senozhatsky@chromium.org
Signed-off-by: Sergey Senozhatsky <senozhatsky@chromium.org>
Reported-by: Chris Mason <clm@meta.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 drivers/block/zram/zram_drv.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/block/zram/zram_drv.c~zram-rename-internal-slot-api-fix
+++ a/drivers/block/zram/zram_drv.c
@@ -191,7 +191,7 @@ static void mark_slot_accessed(struct zr
 	clear_slot_flag(zram, index, ZRAM_IDLE);
 	clear_slot_flag(zram, index, ZRAM_PP_SLOT);
 #ifdef CONFIG_ZRAM_TRACK_ENTRY_ACTIME
-	zram->table[index].attr.ac_time = ktime_get_boottime();
+	zram->table[index].attr.ac_time = (u32)ktime_get_boottime_seconds();
 #endif
 }
 
_
